src/helpers/BapiClient.ts

```diff
@@ -477,9 +483,17 @@ export class BapiClient {
       );
 
       for (const itemToAdd of items) {
-        const existingBasketItem = client.latestBasket.items.find(
-          item => item.variant.id == itemToAdd.variantId,
-        );
+        const existingBasketItem = client.latestBasket.items.find(item => {
+          if (item.variant.id != itemToAdd.variantId) {
```

Comment:
```suggestion
          if (item.variant.id !== itemToAdd.variantId) {
```

Reply:
Previous code also does shallow check, so was trying to be on the safe side here and keep it same